src/core/flexible_hvac.jl

```diff
@@ -50,11 +63,9 @@ be paid.
 There are two construction methods for `FlexibleHVAC`, which depend on whether or not the data was 
 loaded in from a JSON file. The issue with data from JSON is that the vector-of-vectors from the JSON 
 file must be appropriately converted to Julia Matrices. When loading in a Scenario from JSON that 
-includes a `FlexibleHVAC` model if you include the `flex_hvac_from_json` argument to the `Scenario` 
+includes a `FlexibleHVAC` model, if you include the `flex_hvac_from_json` argument to the `Scenario` 
 constructor then the conversion to Matrices will be done appropriately. For example:
```

Comment:
@NLaws is this missing the example following "For example:"?

Reply:
Yes, it is fixed in a separate branch but you can remove it here.